lib/archethic_web/graphql_schema/resolver.ex

```diff
@@ -33,6 +34,54 @@ defmodule ArchethicWeb.GraphQLSchema.Resolver do
     end
   end
 
+  def get_token(address) do
+    genesis_address =
+      case TransactionChain.fetch_genesis_address_remotely(address) do
+        {:ok, genesis_address} -> genesis_address
+        _ -> nil
```

Comment:
if you are not getting the genesis address, this is a network issue, so we should handle this